contracts/amm/pool/JosephInternal.sol

```diff
@@ -50,6 +50,9 @@ abstract contract JosephInternal is
     uint32 internal _maxLiquidityPoolBalance;
     uint32 internal _maxLpAccountContribution;
 
+    /// @dev The threshold for auto-rebalancing the pool. Assume x1000 in calculation.
```

Comment:
What decimals do you use in this place?
What does this means, 'Assume x1000 in calculation'?

Reply:
Without decimals. The value represents multiples of 1000. I updated comments